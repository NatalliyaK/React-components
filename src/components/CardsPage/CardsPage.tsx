import React, { Component } from 'react';
import SearchPanel from '../SearchPanel';

export default class CardsPage extends Component {
  render() {
    return (
      <>
        <SearchPanel />
      </>
    );
  }
}
