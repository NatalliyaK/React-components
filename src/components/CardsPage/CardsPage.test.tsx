import { render } from '@testing-library/react';
import CardsPage from './CardsPage';

describe('CardsPage component', () => {
  it('render correctly', () => {
    const { asFragment } = render(<CardsPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
