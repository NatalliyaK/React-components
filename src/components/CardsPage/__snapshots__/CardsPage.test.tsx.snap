// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardsPage component render correctly 1`] = `
<DocumentFragment>
  <form
    class="search"
  >
    <input
      class="search-control"
      placeholder="Search"
      type="text"
      value=""
    />
  </form>
  <div
    class="card-list"
  >
    <div
      class="card-list__container"
    >
      <img
        class="picture"
        src="./image/img1.jpg"
      />
      <h1
        class="title"
      >
        picture1
      </h1>
      <p
        class="description"
      >
        exemple1
      </p>
      <div
        class="control"
      >
        <button
          type="button"
        >
          <i
            class="fa fa-star"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-trash"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-heart"
          />
        </button>
      </div>
    </div>
    <div
      class="card-list__container"
    >
      <img
        class="picture"
        src="./image/img2.jpg"
      />
      <h1
        class="title"
      >
        picture2
      </h1>
      <p
        class="description"
      >
        exemple2
      </p>
      <div
        class="control"
      >
        <button
          type="button"
        >
          <i
            class="fa fa-star"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-trash"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-heart"
          />
        </button>
      </div>
    </div>
    <div
      class="card-list__container"
    >
      <img
        class="picture"
        src="./image/img3.jpg"
      />
      <h1
        class="title"
      >
        picture3
      </h1>
      <p
        class="description"
      >
        exemple3
      </p>
      <div
        class="control"
      >
        <button
          type="button"
        >
          <i
            class="fa fa-star"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-trash"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-heart"
          />
        </button>
      </div>
    </div>
    <div
      class="card-list__container"
    >
      <img
        class="picture"
        src="./image/img4.jpg"
      />
      <h1
        class="title"
      >
        picture4
      </h1>
      <p
        class="description"
      >
        exemple4
      </p>
      <div
        class="control"
      >
        <button
          type="button"
        >
          <i
            class="fa fa-star"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-trash"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-heart"
          />
        </button>
      </div>
    </div>
    <div
      class="card-list__container"
    >
      <img
        class="picture"
        src="./image/img5.jpg"
      />
      <h1
        class="title"
      >
        picture5
      </h1>
      <p
        class="description"
      >
        exemple5
      </p>
      <div
        class="control"
      >
        <button
          type="button"
        >
          <i
            class="fa fa-star"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-trash"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-heart"
          />
        </button>
      </div>
    </div>
    <div
      class="card-list__container"
    >
      <img
        class="picture"
        src="./image/img6.jpg"
      />
      <h1
        class="title"
      >
        picture6
      </h1>
      <p
        class="description"
      >
        exemple6
      </p>
      <div
        class="control"
      >
        <button
          type="button"
        >
          <i
            class="fa fa-star"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-trash"
          />
        </button>
        <button
          type="button"
        >
          <i
            class="fa fa-heart"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    class="form"
  >
    <form
      class="form-container"
    >
      <div>
        <label
          class="form-item"
          for="name"
        >
          Name:
          <input
            class="form-item__element"
            name="name"
            type="text"
          />
        </label>
      </div>
      <div>
        <label
          class="form-item"
          for="surname"
        >
          Surname:
          <input
            class="form-item__element"
            name="surname"
            type="text"
          />
        </label>
      </div>
      <div>
        <label
          class="form-item"
          for="dataBirth"
        >
          Birth data:
          <input
            class="form-item__element"
            type="date"
          />
        </label>
      </div>
      <div>
        <label
          class="form-item"
          for="country"
        >
          Country:
          <select
            name="country"
          >
            <option
              value="Belarus"
            >
              Belarus
            </option>
            <option
              value="Russia"
            >
              Russia
            </option>
            <option
              value="Ukraine"
            >
              Ukraine
            </option>
          </select>
        </label>
      </div>
      <div>
        <label
          class="form-item"
          for="gender"
        >
          Sex
          <input
            name="gender"
            type="radio"
            value="male"
          />
           Male
          <input
            name="gender"
            type="radio"
            value="female"
          />
           Female
        </label>
      </div>
      <div>
        <label
          class="form-item"
          for="agree"
        >
          agree to the processing of data
          <input
            name="agree"
            type="checkbox"
          />
        </label>
      </div>
      <div
        class="check"
      >
        <input
          type="submit"
        />
      </div>
    </form>
  </div>
</DocumentFragment>
`;
